generate/generate.go

```diff
@@ -266,40 +261,15 @@ func (g *Generator) generatePlugin(req plugin.CodeGeneratorRequest, gen config.G
 }
 
 func (g *Generator) generatePluginGo(req plugin.CodeGeneratorRequest) error {
-	req.Parameter = proto.String("plugins=grpc")
-	bs, err := proto.Marshal(&req)
-	if err != nil {
-		return err
-	}
-
-	cmd := exec.Command("protoc-gen-go")
-	cmd.Stdin = bytes.NewReader(bs)
-	cmd.Stderr = os.Stderr
-	out, err := cmd.Output()
-	if err != nil {
-		return fmt.Errorf("error executing protoc-gen-go: %s, %v", out, err)
-	}
-	var resp plugin.CodeGeneratorResponse
-	if err := proto.Unmarshal(out, &resp); err != nil {
-		return err
-	}
-	for _, rf := range resp.File {
-		// to turn foo.gunk.pb.go into foo.pb.go
-		dir, f := filepath.Split(*rf.Name)
-		outPath := strings.Replace(g.origPaths[dir]+f, ".gunk", "", 1)
-		data := []byte(*rf.Content)
-		if err := ioutil.WriteFile(outPath, data, 0644); err != nil {
-			return fmt.Errorf("unable to write to file %q: %v", outPath, err)
-		}
-	}
-	return nil
+	return g.generatePlugin(req, config.Generator{
+		Command: "protoc-gen-go",
+		Params:  []config.KeyValue{{"plugins", "grpc"}},
+	})
```

Comment:
@vishen I noticed that generatePluginGo was now an almost exact copy of generatePlugin, so I took the liberty of unifying them to avoid updating the same code twice.

In the future, I presume we'll want to remove this function entirely, and have some config loading logic, where we treat a missing config as only running a predefined set of generators. `protoc-gen-go` with this parameter would be one of them.